src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.Designer.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.Windows.Forms;
+using System.Windows.Forms;
```

Comment:
Undo changes from this file. We generally avoid whitespace-only changes to files, and updating the byte order mark is included.

Reply:
good catch.